### PR TITLE
feat(Table): Added zebra stripes to rows

### LINKED ISSUE #424: ErrorPage: CommonProps are missing
Need to add `CommonProps` to `ErrorPage` component.
Not related, but is the image not too small?

### LINKED ISSUE #439: Table: zebra rows
image

### PR DESCRIPTION
**Pull request by veekeys**
Added zebra stripes class. Surprisingly, works with sub rows too.
Visual tests did not fail though.. So just manually updated the image.
Added prop `isStriped` so up for discussion for the name and description.

Closes #439 

image

### CODE REVIEW

**Comment by bentleyvk** [comment]
Have you tested properly? 😁 Because it needs some CSS fixes:
zebra-table

**Comment by veekeys** [comment]
> Have you tested properly? 😁 Because it needs some CSS fixes: zebra-table

Fixed through css.

File: src/core/Table/Table.tsx [hunk 1]
```diff
@@ -213,6 +213,11 @@ export type TableProps<
    * @default false
    */
   isResizable?: boolean;
+  /**
+   * Use to have zebra-striped table rows.
+   * @default false
+   */
+  isStriped?: boolean;
```
Comment (gretanausedaite) [inline src/core/Table/Table.tsx:220]: Maybe something like `styleType` : default or striped would be better? This way we could add more styles later.

**Review by gretanausedaite — COMMENTED** [review]
I think separate story for striped table would be better. I know we have like a million stories for table but I'd never look into paginator story for style (striped vs default).

File: src/core/Table/Table.tsx (hunk 1 above)
Comment (veekeys) [reply src/core/Table/Table.tsx]: Changed prop and added separate story

Comment (mayank99) [reply src/core/Table/Table.tsx]: Might be better to do `zebra-rows`. we could add zebra columns in the future

**Review by mayank99 — APPROVED** [review]

**Pull request merged**